### ISSUE TITLE
stiling issue with site escape metric

### ISSUE THREAD

**Issue by Bernadetadad** [dms-vep/Flu_H5_American-Wigeon_South-Carolina_2021-H5N1_DMS#95]
Something strange about how 'sum' is displayed for 'site escape metric'
<img width="768" alt="image" src="https://github.com/dms-vep/Flu_H5_American-Wigeon_South-Carolina_2021-H5N1_DMS/assets/22810662/62138c83-b49b-40ce-a4e3-df68194728c4">

**Comment by jbloom** [comment]
Yes, I think @WillHannon-MCB is still looking into the fonts for the `altair` sliders, but at least they show up now.

**Comment by WillHannon-MCB** [comment]
Weird, I thought I fixed this on the most recent pull request, but clearly not for all browsers. I'll fix this!
